test/helpers/events.js

```diff
@@ -18,7 +18,7 @@ const fireCustomEvent = function (el, event, data) {
 	}));
 };
 
-module.exports = {
-	fireEvent: fireEvent,
-	fireCustomEvent: fireCustomEvent
+export {
```

Comment:
Because these are now named exports, the tests will need to import them as named imports. I.E. `import {fireEvent} from `test/helpers/events.js`;